kolibri_instant_schools_plugin/management/commands/migratev1database.py

```diff
@@ -0,0 +1,188 @@
+import hashlib
+
+from django.conf import settings
+from django.core.management import call_command
+from django.core.management.base import BaseCommand, CommandError
+from django.utils import timezone
+
+from kolibri.auth.constants.role_kinds import ADMIN
+from kolibri.auth.constants.facility_presets import mappings
+from kolibri.auth.models import Facility, FacilityUser, Role
+from kolibri.core.device.models import DevicePermissions, DeviceSettings
+from kolibri.logger.models import ContentSummaryLog, ContentSessionLog, MasteryLog, UserSessionLog, AttemptLog
+from ...models import PhoneToUsernameMapping
+
+from sqlalchemy import MetaData, create_engine
+from sqlalchemy.ext.automap import automap_base
+from sqlalchemy.orm import sessionmaker
+
+USER_ID_MAPPING = {}
+SUMMARYLOG_ID_MAPPING = {}
+MASTERYLOG_ID_MAPPING = {}
+SESSIONLOG_ID_MAPPING = {}
+
+class Command(BaseCommand):
+    help = "Migrate data from the old Instant Schools v1 database into the v2 database."
+
+    requires_system_checks = False
+
+    def handle(self, *args, **options):
+
+        # ensure the new databases have both been migrated
+        call_command("migrate")
```

Comment:
Without it, I was getting issues. But I think it was because of the migration that I now elided, so it may not be needed anymore -- but yeah, doesn't hurt, and _makes sure_.

Reply:
Tried without again, and it seems it still errors out... not sure precisely what order it does things in, but it doesn't seem to have migrated the primary DB by the time it enters this command.

```
  File "/home/jamalex/fle/kolibri-instant-schools-plugin/kolibri_instant_schools_plugin/management/commands/migratev1database.py", line 52, in handle
    new_facility, _ = Facility.objects.get_or_create(name=old_facility.name)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/manager.py", line 122, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/query.py", line 465, in get_or_create
    return self.get(**lookup), False
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/query.py", line 381, in get
    num = len(clone)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/query.py", line 240, in __len__
    self._fetch_all()
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/query.py", line 1074, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/query.py", line 52, in __iter__
    results = compiler.execute_sql()
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 848, in execute_sql
    cursor.execute(sql, params)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/utils.py", line 95, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/jamalex/.virtualenvs/kolibri/local/lib/python2.7/site-packages/django/db/backends/sqlite3/base.py", line 323, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: kolibriauth_collection
```